pkgs/development/python-modules/mkdocs-autolinks-plugin/default.nix

```diff
@@ -8,15 +8,15 @@
 
 buildPythonPackage rec {
   pname = "mkdocs-autolinks-plugin";
-  version = "0.7.1";
+  version = "071";
   pyproject = true;
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "zachhannum";
     repo = "mkdocs-autolinks-plugin";
     # The commit messages mention version 0.7.1, but the tag is v_071.
-    rev = "e2b649eb4db23459bcec121838f27c92c81f9ce1";
+    rev = "refs/tags/v_${version}";
```

Comment:
I think this is a good change, and we could use it even if we don't change the version number. The comment above would already clarify the situation, and I think pointing to a tag makes things clearer than to a commit hash.